.github/workflows/python-package-conda.yml

```diff
@@ -28,16 +24,16 @@ jobs:
           python-version: 3.9
           mamba-version: "*"
           activate-environment: donkey
-          environment-file: ${{matrix.ENV_FILE}}
           auto-activate-base: false
           channels: default, conda-forge, pytorch
           channel-priority: true
       - name: Conda info and list
         run: |
-          echo Environment file ${{matrix.ENV_FILE}}
           conda info
           conda list
       - name: Install donkey
-        run: pip install -e .[pc]
+        run: | 
+          pip install -e .[pc,dev]
```

Comment:
should this list include `macos`?

Reply:
Not yet. I will work on a dedicated apple silicon CI run afterwards. For now we only test on apple x86.